controllers/user.js

```diff
@@ -5,6 +5,45 @@ const bcrypt = require("bcryptjs");
 const jwt = require("jsonwebtoken");
 
 module.exports = {
+  signUp: async function (req, res) {
+    try {
+      const { name, email, password, profileUrl } = req.body;
+      if (!name && !email && !password && !profileUrl) {
```

Comment:
```suggestion
      if (!name && !email && !password) {
```
We have to remove the `profileUrl` field from required list. It is optional.